src/sections/Meshery/Features-Col/data/index.js

```diff
@@ -1,148 +1,189 @@
+        className: "--on",
+      },
+      {
+        content:
+          "Import your existing Grafana dashboards. Ingest any metric from Prometheus",
+        className: "--on",
+      },
+      {
+        content:
+          "Create and share performance profiles and test results with team members.",
+        className: "--on",
+      },
+      {
+        content: "Identify the MeshMark of a specific network function",
+        className: "--on",
+      },
+      {
+        content: "Baseline and track deltas of your workload performance",
+        className: "--on",
+      },
+    ],
+    count: {
+      value: 0,
+      description: "performance tests run",
     },
-    {
-      content: "Baseline and track deltas of your workload performance",
-      className: "--on",
-    }
-  ],
-  count: {
-    value: 0,
-    description: "performance tests run",
-  },
-};
+  };
+
+  // const collaborationFeatureData = {
+  //   name: "Performance",
+  //   description: "Assess the performance of your service mesh",
+  //   link: "#",
+  //   btnText: "Get Started",
+  //   services: [
+  //     {
+  //       content:
+  //         "Distributed performance analysis supporting 3 different types of load generators",
+  //       className: "--on",
+  //     },
+  //     {
+  //       content: "Built-in statistical analysis of service latency quartiles",
+  //       className: "--on",
+  //     },
+  //     {
+  //       content: "Integration with Prometheus and Grafana",
+  //       className: "--on",
+  //     },
+  //     {
+  //       content: "Baseline and track deltas of workload performance.",
+  //       className: "--on",
+  //     },
+  //   ],
+  //   count: {
+  //     value: 0,
+  //     description: "performance tests run",
+  //   },
+  // };
+
+  const data = {
+    features: [
+      lifecycleFeatureData,
+      configurationFeatureData,
+      performanceFeatureData,
+      // collaborationFeatureData,
+    ],
+  };
 
-// const collaborationFeatureData = {
-//   name: "Performance",
-//   description: "Assess the performance of your service mesh",
-//   link: "#",
-//   btnText: "Get Started",
-//   services: [
-//     {
-//       content:
-//         "Distributed performance analysis supporting 3 different types of load generators",
-//       className: "--on",
-//     },
-//     {
-//       content: "Built-in statistical analysis of service latency quartiles",
-//       className: "--on",
-//     },
-//     {
-//       content: "Integration with Prometheus and Grafana",
-//       className: "--on",
-//     },
-//     {
-//       content: "Baseline and track deltas of workload performance.",
-//       className: "--on",
-//     },
-//   ],
-//   count: {
-//     value: 0,
-//     description: "performance tests run",
-//   },
-// };
+  // Count the total number of integrations
+  const totalIntegrations =
+    Math.ceil(integrations.allMdx.totalCount / 10) * 10;
```

Comment:
What's the purpose of doing this?

Reply:
I am doing this because the total number is somewhere less than 150 as far I know, so making it closest to an integer as it is on home page